documentation/markdown/accesories/aluminium-case/user_guide/user_guide.md

```diff
@@ -0,0 +1,106 @@
+<center>
+
+![Fin logo](../../../assets/balenaFin_logo.png)
+
+| **Document Type** | User Guide |
+| --- | --- |
+| **Product ID:** | TBD |
+| **Product Name** | Modular DIN-rail Case |
+| **Document Version** | 0.0.1 |
+| **Author** | Carlo Maria Curinga |
+| **State (Draft/Proposed/Approved)** | Draft |
+
+</center>
+
+## Revision history
+
+<center>
+
+| **Date (dd/mm/yyyy)** | **Version** | **Author** | **Description** |
+| --- | --- | --- | --- |
+| 14/01/2020 | 0.0.1 | Carlo Maria Curinga | First release |
+
+</center>
+
+<div class="page-break"></div>
+
+# 1. Introduction
+
+![Fin aluminium case](./pictures/assembled.jpg)
+
+The Aluminium case is the official "heavy duty" case for the balenaFin. It exposes the following ports to the outside :
+
+- 2 x USB
+- Ethernet (RJ45)
+- HDMI
+- Barrel jack power connector
+- 2 x External antenna mounting points
+
+# 2. What's in the box?
+
+1. Aluminium Case for balenaFin  
+    1.1 - bottom  
+    1.2 - Lid  
+    1.3 - left side  
+    1.4 - right side
+    1.5 - front side
+    1.6 - rear side
+2. M3 standoffs
+3. M3 screws
+4. 10mm x 10mm thermal pad
+
+![Fin block diagram](./pictures/components.jpg)
+
+<div class="page-break"></div>
+
+# 3. Assembly instructions
+
+## 3.1 Attaching the balenaFin
+
+**Items needed**
+- balenaFin board
+- 4 x M3 standoffs (3 if HAT is mounted)
+- Case Lid
+- 10mm x 10mm thermal pad
+
+**Instructions**
+- Place the bottom part of the case on the table with the inside facing upwards.
```

Comment:
When you say "bottom part", it might be useful to add the reference (1.1 in this case). It makes it easy to identify